client/app/pages/queries/components/QueryPageHeader.jsx

```diff
@@ -120,14 +120,7 @@ export default function QueryPageHeader({
             title: "Unpublish",
             onClick: unpublishQuery,
           },
-        },
-        {
-          showAPIKey: {
-            isAvailable: !clientConfig.disablePublicUrls && !queryFlags.isNew,
-            title: "Show API Key",
-            onClick: openApiKeyDialog,
-          },
```

Comment:
@DebangshuB comment it out instead of removing it, also mention the JIRA ticket no. in comment